vernac/comInductive.ml

```diff
@@ -328,14 +327,17 @@ let check_param = function
 | CLocalPattern {CAst.loc} ->
     Loc.raise ?loc (Stream.Error "pattern with quote not allowed here")
 
-let interp_mutual_inductive (paramsl,indl) notations cum poly prv finite =
+let interp_mutual_inductive_gen env0 (uparamsl,paramsl,indl) notations cum poly prv finite =
   check_all_names_different indl;
   List.iter check_param paramsl;
-  let env0 = Global.env() in
+  if not (List.is_empty uparamsl) && not (List.is_empty notations)
+  then user_err (str "Inductives with uniform parameters may not have attached notations.");
```

Comment:
May I ask if there is any deep problem here, or it is just lack of time? @herbelin do you think it is hard to support this?

Reply:
Hard to support notations? If the notation for the type involves at least one of the arguments that would not be mentioned as in e.g. the case of `\/`, or `/\`, the notation would stop to make sense with this at least one argument not being mentioned.